reinforced_lib/exts/gymnasium.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any
+
+import gymnasium as gym
+import numpy as np
+
+from reinforced_lib.exts import BaseExt, observation, parameter
+
+
+class Gymnasium(BaseExt):
```

Comment:
I still do not know why do we need this wrapper :(

Reply:
It's not super useful, but it does make it a bit easier to use the agent with the gym environments. Instead of writing:
```python
state, reward, terminal, truncated, info = env.step(action)
action = rl.sample(
      update_observations={'env_state': state, 'action': action, 'reward': reward, 'terminal': terminal},
      sample_observations={'env_state': state}
)
```

we can simply write:
```python
action = rl.sample(*env.step(action))
```